### ISSUE TITLE
Investigate why avant anti virus blocks enabling min vid

### ISSUE THREAD

**Issue by gregcbrown** [meandavejustice/min-vid#360]
Attempting to enable Min Vid:
"Uh oh. ⁨Min Vid⁩ could not be enabled. Try again later."

**Comment by SoftVision-PaulOiegas** [comment]
@gregcbrown Hi, can you please navigate to "about:addons" page and check for Test Pilot update before attempting to re-enable Min Vid ? It seams that the Test Pilot page fails to enable the new experiments launched until the browser checks for updates and installs the latest version of Test Pilot. 

If this does not fix your problem, is it possible that you have a custom firewall installed or a super protective antivirus client ? We had reporters in the past that encountered problems related to this causes and everything was resolved if the firewall was bypassed or if some antivirus permissions were added for Test Pilot & experiments.

**Comment by gregcbrown** [comment]
Thanks Paul,

I did as suggested and no help. I’m using standard windows 10 firewall and AVAST antivirus.

 

Thanks,

Greg

**Comment by SoftVision-PaulOiegas** [comment]
I would suggest to temporarily disable the Avast antivirus to see if it has something related. But that's up to you. 
Also, can you open the browser console before trying to enable Min Vid and attach here a document with the logs after you attempt to enable it ? Those might help the developers to find and fix for the issue.

**Comment by gregcbrown** [comment]
Thanks Paul, I temporarily disabled Avast and it enabled.

 

Greg

**Comment by SoftVision-PaulOiegas** [comment]
Good to hear that Greg. Enjoy the experiment and please report any issues you find so we can make it better.

@meandavejustice is there a chance that we did something wrong in the code that makes Avast AV to block the experiment to be enabled ?

**Comment by meandavejustice** [comment]
@SoftVision-PaulOiegas I'm not very familiar with Avast, we can reopen and rename this issue to investigate why it could be blocked. I wonder if this is an issue for all testpilot addons.

**Comment by SoftVision-PaulOiegas** [comment]
@meandavejustice It could be a general Test Pilot issue since it's like the 3rd or 4th issue that was fixed with bypassing the firewall or antivirus on different experiments. I raised this possible issue in one of our calls but I don't think someone investigated this since then.

**Comment by rugk** [comment]
AFAIk Avast does crappy things anyway with Firefox add-ons. See https://www.eff.org/privacybadger#faq-I%27m-having-a-problem-installing-the-browser-extension. e.g.

**Comment by meandavejustice** [comment]
@SoftVision-PaulOiegas I think you are right. We should maybe detect Avant on the testpilot side and ask people to disable it when installing like in the eff link that @rugk linked to above.

**Comment by rugk** [comment]
I don't know what Avast does, but [based on this thread](https://forum.avast.com/index.php?topic=189448.0) I assume that it blocks installations from third-party websites (not AMO). If this is true all test pilot experiments would be affected.

**Comment by gregcbrown** [comment]
This might be of interest to you. This just popped up:

 



 

From: Greg [mailto:gregory.c.brown@gmail.com] 
Sent: Friday, September 30, 2016 11:07 AM
To: 'meandavejustice/min-vid' <reply@reply.github.com>; 'meandavejustice/min-vid' <min-vid@noreply.github.com>
Cc: 'Mention' <mention@noreply.github.com>
Subject: RE: [meandavejustice/min-vid] Cannot enable Min Vid (#360)

 

Thanks Paul, I temporarily disabled Avast and it enabled.

 

Greg

**Comment by SoftVision-PaulOiegas** [comment]
Dave, after we tested a bit yesterday with Avast antivirus, we found out that the client installs by default an extension into Firefox that works a bit like tracking protection. It blocks by default the Google Analitics pings. If you disable the restriction the installation works like a charm. 
So it seams that not the entire AV client needs to be disabled, just the restriction switch for this to work.
